modules/private-clusters-setting-dns-private.adoc

```diff
@@ -4,13 +4,22 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="private-clusters-setting-dns-private_{context}"]
-= Setting DNS to private
+= Configuring DNS records to be published in a private zone
 
-After you deploy a cluster, you can modify its DNS to use only a private zone.
+For all {product-title} clusters, whether public or private, DNS records are published in a public zone by default.
+
+You might use a public zone for DNS records in cases where you want to have a private cluster while still allowing clients to look up names for applications running on that cluster. For example, if an organization has machines that connect to the public internet, and establish VPN connections for certain private IP ranges in order to connect to private services. The DNS lookups from these machines use the public DNS to determine the private addresses of those services, and then connect to the private addresses over the VPN.
+
+Conversely, you might use a private zone for DNS records to avoid exposing sensitive information contained in DNS records to the public. DNS records can include information such as internal domain names, internal IP addresses, or the number of clusters at an organization. You might also simply have no need to publish records publicly and want to avoid unnecessarily publishing DNS records.
+
+If all the clients that need to connect to services within the cluster use a private DNS service that has the DNS records from the private zone, then there is no need to have a public DNS record for the cluster.
```

Comment:
I think the original looked more like this.  I edited it to fit some of the prose you used.

Remove the public zone from the cluster DNS config to avoid exposing DNS records to the public. You may want to avoid exposing sensitive information (internal domain names, internal IP addresses, or the number of clusters at an organization), or you might simply have no need to publish records publicly. If all the clients that should be able to connect to services **within** the cluster use a private DNS service that has the DNS records from the private zone, then there is no need to have a public DNS record for the cluster.